src/Globe/Globe.js

```diff
@@ -187,9 +186,11 @@ Globe.prototype.setLayerVisibility = function(id, visible) {
     this.layersConfiguration.setLayerVisibility(id, visible);
 
     var cO = function(object) {
```

Comment:
There's error in [Globe.prototype.setLayerVisibility](https://github.com/iTowns/itowns2/blob/29b3bfcf2b63c22ead626b8f0b5c0ab88bbecb7c/src/Globe/Globe.js#L181)

Do not traverse all nodes when changing the visibility of a layer. 

Do like this (not tested)
```js
Globe.prototype.setLayerVisibility = function(id, visible) {
    this.layersConfiguration.setLayerVisibility(id, visible);
    var featureLayer = this.getFeatureLayerByName(id);
    if(featureLayer) {
        // feature layer hide only root node
        featureLayer.children[0].visible = visible;
    } else {
        // imagery layer
        var cO = function(object) {
        if (object.material.setLayerVisibility) {
             object.material.setLayerVisibility(object.getIndexLayerColor(id), visible);
        }
        this.tiles.children[0].traverse(cO);
   }
};
```